packages/notifi-dapp-example/src/app/notifi/dashboard/page.tsx

```diff
@@ -12,10 +13,38 @@ export default function NotifiDashboard() {
   return (
     <div className="min-h-screen flex flex-col items-center justify-center">
       Dummy Dashboard {JSON.stringify({ isAuthenticated })}
+      <button
```

Comment:
This is just an temp utility to test expiry page. will be removed when implementing Dashboard